tools/roll.py

```diff
@@ -29,7 +35,7 @@ def main():
     path = path[1:]
 
   depdir = args.path
-  if depdir == None:
+  if depdir is None:
```

Comment:
It's a good habit to do `is None` not `== None` because classes can override `__eq__` and not deal with `None`, in theory.